### PR TITLE
Docs - Edit Tutorial

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Edited Stardust tutorial:

* Split tutorial.md into 10 steps
* Added references and links
* Updated sidebar
* Made the user the center
* General style edits

## Type of change

- Documentation enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Wiki was built locally

## Change checklist

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

## To Do

- [ ] Add meta tags
- [ ] Review edits
- [ ] Test Code 
- [ ] Add "Putting it all together" per section
- [ ] Add "Putting it all together" files in static dir so users can download
- [ ] Peer review

### CODE REVIEW

File: documentation/docs/tutorials/value-transactions/01-introduction.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Stardust for iota.js: Value Transactions
+
+In this series, you will This series of how-to pages is intended to Javascript developers that want to get familiar
```
Comment (jmcanterafonseca-iota) [inline documentation/docs/tutorials/value-transactions/01-introduction.md]: check phrase

File: documentation/docs/tutorials/value-transactions/03-generate-a-seed.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Generate Seeds
+
+The first thing you will to send or receive a transaction is a valid address that can unlock outputs or
+receive new outputs. Addresses are derived from a public key and their associated outputs can be unlocked with the
+corresponding private key. As with Firefly wallets, you can generate multiple addresses from an initial master secret 
+(*seed phrase*), and, from that point on, use a hierarchical deterministic method to derive multiple addresses.
+
+## Generate a Seed Phrase and Ed25519 Seed
```
Comment (jmcanterafonseca-iota) [inline documentation/docs/tutorials/value-transactions/03-generate-a-seed.md]:
```suggestion
## Generate a Seed Phrase and Ed25519 Master Seed
```

File: documentation/docs/tutorials/value-transactions/08-transfer-funds.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+const sourceAddressPublicKey = "0x5782872d...";
+const sourceAddressPrivateKey = "0x003dd7e...";
+
+const destAddress = "0xbc9a935...";
+```
+
+You will need to take the following steps to transfer funds:
+
+1. [Prepare the input](#prepare-the-input) by selecting the output to be consumed and turn into a transaction input.
+3. [Prepare the outputs](#prepare-the-outputs)
```
Comment (jmcanterafonseca-iota) [inline documentation/docs/tutorials/value-transactions/08-transfer-funds.md]:
```suggestion
2. [Prepare the outputs](#prepare-the-outputs)
```

**Comment by lucas-tortora** [comment]
@jmcanterafonseca-iota , please check the "Putting it all together" sections and the zip file containing the code sample.  If possible, it'd be great to get a valid private and public key for the `send-value-transaction.ts` so users can simply run it without any edits and see results. 

It may be a good idea to include a sample package.json and tsconfig.json as well to make sure users don't have any config issues, wdyt?

**Comment by Dr-Electron** [comment]
Hi @lucas-tortora what's the idea here? Do you want to have this tutorial linked in the tutorial section as one big tutorial?
Because you are still missing the tutorial config for that 🤔

**Comment by lucas-tortora** [comment]
> Hi @lucas-tortora what's the idea here? Do you want to have this tutorial linked in the tutorial section as one big tutorial? Because you are still missing the tutorial config for that thinking

I wanted your review regarding content, structure, etc. But indeed, the idea is to link to this tutorial from the tutorial section. Looking at the existing tutorials, it's not quite clear to me how to do so. 

Should I add an entry to https://github.com/iota-wiki/iota-wiki/blob/main/tutorials/single-page-tutorials/docusaurus.config.js to get this listed in the tutorial section? Or should I add it to the repo's docusarus.config.js?

File: documentation/docs/tutorials/value-transactions/01-introduction.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+description: "Get acquainted with the structure and functionality of transactions in IOTA's Stardust protocol in the
+Shimmer network using iota.js primitives to issue value transactions."
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/value-transactions/01-introduction.md:3]:
```suggestion
Shimmer network by using iota.js primitives to issue value transactions."
```

File: documentation/docs/tutorials/value-transactions/01-introduction.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+description: "Get acquainted with the structure and functionality of transactions in IOTA's Stardust protocol in the
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/value-transactions/01-introduction.md:2]:
```suggestion
description: "Learn the structure and functionality of transactions in IOTA's Stardust protocol in the
```

A slightly shorter version.

File: documentation/docs/tutorials/value-transactions/01-introduction.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- shimmer
+- ledger
+- outputs
+- inputs
+
+---
+
+# Stardust for iota.js: Value Transactions
+
+This tutorial will guide you as you get acquainted with the structure and functionality of transactions in
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/value-transactions/01-introduction.md]:
```suggestion
This tutorial will get you acquainted with the structure and functionality of transactions in
```

File: documentation/docs/tutorials/value-transactions/01-introduction.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+IOTA's [Stardust protocol](https://wiki.iota.org/shimmer/introduction/welcome) in
+the [Shimmer network](https://shimmer.network) using [iota.js](https://github.com/iotaledger/iota.js) primitives to
+issue value transactions.
+
+## Introduction
+
+Starting from [Chrysalis](https://wiki.iota.org/introduction/welcome), IOTA is
+a [UTXO-based (Unspent Transaction Output)](https://wiki.iota.org/introduction/reference/details#unspent-transaction-output-utxo)
+Ledger. Each UTXO, also known as  **output**, has an associated number of tokens that determines its value.
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/value-transactions/01-introduction.md:28]:
```suggestion
a [UTXO-based](https://wiki.iota.org/introduction/reference/details#unspent-transaction-output-utxo)
Ledger, where UTXO means *unsepnt transaction output*, or just *output* for short. Each UTXO has an associated number of tokens that determines its value.
```

File: documentation/docs/tutorials/value-transactions/01-introduction.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+Thus, the permanent data on the ledger is composed of a set of records (*outputs*) that can be unlocked by the owner of
+its associated address, i.e., the one who knows the address' private key.
+
+There are different [output types](https://wiki.iota.org/shimmer/learn/outputs). This tutorial series will only focus on
+traditional value outputs known as [`IBasicOutput`](./../../references/client/interfaces/IBasicOutput.md). Outputs are
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/value-transactions/01-introduction.md]: I don't think that "traditional" is a right word here.

File: documentation/docs/tutorials/value-transactions/01-introduction.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+## Storage Costs and Deposits
+
+UTXOs need to be stored by [Hornet Nodes](https://wiki.iota.org/shimmer/hornet/welcome), so there is a storage cost.
+The Stardust protocol defines a parameter, named ["vByte cost"](../../references/client/interfaces/IRent#vbytecost),
+that establishes the cost of storage of each "virtual byte" in *Glow* (one Glow corresponds to 0.000001 Shimmer).
+As each output consumes a number of vBytes in the Ledger, it is a necessary condition for an output
+to cover the vByte cost with its value in the Ledger. This means that Nodes will reject transactions
+whose outputs do not meet this condition. For instance, in the current Shimmer network, you cannot
+send `1 Glow` to a new output because the [storage deposit cost](09-understanding-deposits.md) of such an output will be
+higher. The size of a UTXO and its cost depends structurally on the type and any additional data it may carry.
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/value-transactions/01-introduction.md:50]: Is this information really relevant for the tutorial? Can't we just link an article from the explanations?

File: documentation/docs/tutorials/value-transactions/02-prepare-the-dev-env.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+      you can spin up your own node to run the PoW remotely.
+
+* Access to a Stardust Node (Hornet 2.0.0). You can use the Shimmer testnet Nodes
+  at [https://api.testnet.shimmer.network](https://api.testnet.shimmer.network).
+* (Optional) A TypeScript compiler and related packages
+
+## (optional) Download the Code
+
+You can [download the tutorials codebase](code_samples/tutorials/value-transactions/iota.js-value-transactions-tutorial.zip)
+to follow while you read the next sections. After you have unpacked the `.rar` file to a directory of your choice.
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/value-transactions/02-prepare-the-dev-env.md]:
```suggestion
to follow while you read the next sections. After you have unpacked the `.zip` file to a directory of your choice.
```

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: documentation/docs/tutorials/value-transactions/01-introduction.md (hunk 5 above)
Comment (lucas-tortora) [reply documentation/docs/tutorials/value-transactions/01-introduction.md:2]: It should be Lean about the structure, which isn't that much shorter.

File: documentation/docs/tutorials/value-transactions/01-introduction.md (hunk 8 above)
Comment (lucas-tortora) [reply documentation/docs/tutorials/value-transactions/01-introduction.md]:
```suggestion
value outputs known as [`IBasicOutput`](./../../references/client/interfaces/IBasicOutput.md). Outputs are
```

File: documentation/docs/tutorials/value-transactions/01-introduction.md (hunk 9 above)
Comment (lucas-tortora) [reply documentation/docs/tutorials/value-transactions/01-introduction.md:50]: I think it is, as the last steps in the tutorial focus on reducing storage deposit costs by sweeping outputs.  That being said, we can always restructure, it's what this PR is about for the most part :)

File: documentation/docs/tutorials/value-transactions/02-prepare-the-dev-env.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+## (optional) Download the Code
+
+You
+can [download the tutorials codebase](https://github.com/iotaledger/iota.js/tree/feat/stardust/packages/iota/examples/shimmer-value-transaction-tutorial)
+to follow while you read the next sections.
+
+:::warning Working Directory
+
+This tutorial use `add working directory here` as a working directory. If you are going to use another directory, you
```
Comment (lucas-tortora) [inline documentation/docs/tutorials/value-transactions/02-prepare-the-dev-env.md]: @jmcanterafonseca-iota , whenever you have a moment, we should update this

**Comment by lucas-tortora** [comment]
closed in favour of https://github.com/iotaledger/iota.js/pull/1050

**Pull request closed without merging**